app/uk/gov/hmrc/ivorchestration/persistence/ReactiveMongoConnector.scala

```diff
@@ -32,7 +32,7 @@ object ReactiveMongoConnector extends MongoConfiguration with Logging {
 
   lazy val mongoConnector: MongoConnector = MongoConnector(mongoConfig.uri)
 
-  logger.debug(s"Reactive Mongo configuration being used: $mongoConnector")
+  logger.info(s"Reactive Mongo configuration being used: $mongoConnector")
```

Comment:
Again, I think that this logging isn't very useful for us, so might as well delete please